content-repo/gendocs.py

```diff
@@ -136,6 +139,35 @@ class DeprecatedInfo(TypedDict, total=False):
     note: str
 
 
+def version_convertor(version: str) -> int:
```

Comment:
i'm against parsing the version to int. as 6.10.10 < 7.0.0. but you conversion to number will be 61010 and 700 which will cause and error. please use SemVer comparison using this https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python